ci-excludes.txt

```diff
@@ -0,0 +1,23 @@
+language: go
```

Comment:
looks like the contents of this file have been mixed up with something else

Reply:
Yeah, this isn't right! Not sure how all this got in here